.travis.yml

```diff
@@ -32,17 +33,57 @@ matrix:
       - HDF5_DIR=$HDF5_CACHE_DIR/$HDF5_VERSION
       - H5PY_ENFORCE_COVERAGE=yes
       os: linux-ppc64le
+    - os: linux
+      python: 3.7
+      virt: vm
+      group: edge
+      arch: arm64
+      env:
+        - TOXENV=py37-test-deps
+        - HDF5_VERSION=1.10.5
+        - HDF5_DIR=$HDF5_CACHE_DIR/$HDF5_VERSION
+        - H5PY_ENFORCE_COVERAGE=yes
+        - CIBW_BUILD: cp3[789]-*
+        - CIBW_BEFORE_BUILD: ./ci/get_hdf5_arm64.sh
```

Comment:
For x86_64 wheels, we make our own Docker image with HDF5 added on top of the manylinux images, so we don't need to rebuild HDF5 on every CI run:

https://github.com/h5py/hdf5-manylinux

We should probably do something similar for this, though I imagine it won't be quite as straightforward. Let me know what you think. I can create a `hdf5-manylinux-arm64` repo if that's useful.

Reply:
I created this PR https://github.com/h5py/hdf5-manylinux/pull/2 to make docker image with HDF5 added on top of the manylinux images for aarch64 but if we need separate repo like you mentioned (`hdf5-manylinux-arm64`) I can add changes for aarch64 to this new repo